src/main/java/org/folio/rest/helper/TransactionSummariesHelper.java

```diff
@@ -0,0 +1,20 @@
+package org.folio.rest.helper;
+
+import static org.folio.rest.util.ResourcePathResolver.ORDER_TRANSACTION_SUMMARIES;
+import static org.folio.rest.util.ResourcePathResolver.resourcesPath;
+
+import io.vertx.core.Context;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import org.folio.rest.jaxrs.model.OrderTransactionSummary;
+
+public class TransactionSummariesHelper extends AbstractHelper {
+
+  public TransactionSummariesHelper(Map<String, String> okapiHeaders, Context ctx, String lang) {
+    super(okapiHeaders, ctx, lang);
+  }
+
+  public CompletableFuture<OrderTransactionSummary> createOrderTransactionSummary(OrderTransactionSummary orderSummary) {
+    return handleCreateRequest(resourcesPath(ORDER_TRANSACTION_SUMMARIES), orderSummary).thenApply(orderSummary::withId);
```

Comment:
I think make sense to check that `numTransactions` grater than 0

Reply:
Good point, I think we'll have to do this in storage also, I'll make a change in the schema so that it need not be validated in multiple places

Reply:
Added the validation in code until https://issues.folio.org/browse/RMB-521 is fixed